src/GuideAtom.stories.tsx

```diff
@@ -16,6 +16,112 @@ export const DefaultStory = (): JSX.Element => {
             title="Wednesday's Hong Kong tips"
             html="<p><b>Happy Valley&nbsp;</b></p><p><b>11.45</b> Happy Good Guys <b>12.15</b> Salto Olimpico <b>12.45</b> Seize The Spirit <b>1.15</b> Allied Agility <b>1.45 </b>Hero Time <b>2.15</b> Simply Fluke <b>2.45</b> Brave King <b>3.15</b> Golden Dash <b>3.50</b> This Is Class</p>"
             credit=""
```

Comment:
Your stories are all of the closed component, which I assume isn't what you want. You probably need to add an optional 'expanded' prop on the component that is used as the default state of the exapanded